understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
+        
+        <p>Thus, following responsive web design best practices can help many websites meet Reflow, the success criterion does not mandate fully responsive web pages. Building complex web interfaces (a.k.a., web apps) will often necessitate not only providing revised presentation, but revised functionality, or both when content is enlarged.</p>
+        
+        <figure>
+            <img src="..." alt="TODO">
+            <figcaption>
+              <p>At larger viewport sizes with a default zoom level, buttons on a website invoke popup menus, with menu items that can invoke sub-menus.  At zoomed in viewport sizes, such menus can pose difficulties for users because the menus are often partially positioned off-screen.</p>
+              <p>Revising the presentation and functionality of these menus, for instance by making them full screen and only presenting a single menu level at a time, can help ensure portions of the menu will not be positioned off-screen and thus difficult, if not impossible, for users to access.</p>
+            </figcaption>
+        </figure>
+          
+         <p>Additionally, it can be common for web application to contain sections of content that meet one of Reflow's exceptions and can be presented in a bi-directional scrolling container within the otherwise single-direction scrolling interface.</p>
+
+        <h4>Horizontally scrolling section within a vertically scrolling page</h4>
+        <p>It is not uncommon for a web page to render a widget, such as a "carousel", "swim lane", or otherwise "horizontally scrolling container" which itself contains sub-sections of content that follow the otherwise esbalished reading direction of the page. Such widgets can meet the requirements of this success criterion, so long as the sub-sections of content can accommodate reflowing to each fit within the required width or height - per the intended reading direction each section represents.</p>
+
+        <figure>
+          <img src="img/carousel-large-screen.png" alt="horizontally scrolling carousel, containing multiple 'panels' of content (four visible in the screenshot).">
+          <figcaption>
+            <p>A website provides a “carousel” or “swim-lane” of different panels of teaser (a.k.a., “lead-in”) content.This content container allows for horizontal scrolling, independent from the vertical scrolling of the primary web page, and the vertical direction of reading of the individual sections of content within this horizontally scrolling container.</p>
```

Comment:
I find this sentence quite long. What about:

```suggestion
            <p>A website provides a “carousel” or “swim-lane” of different panels of teaser (a.k.a., “lead-in”) content. This container scrolls horizontally, independent of the vertical scrolling of the primary web page. Each individual panel within the carousel is designed to be read vertically, like the primary content, even as the container scrolls horizontally.</p>
```